kubernetes/README.md

```diff
@@ -180,6 +192,179 @@ select * from taxi_trips_customized limit 10;
 Time: 0.01968175 seconds. 10 rows.
 ```
 
+## Enable Verbose Logging (optional)
```

Comment:
I don't think we should have verbose logging as a standard in basic cookbook recipes.

Reply:
This is an optional step showing how to adjust tracing level for troubleshooting, default configuration and basic steps have default tracing configuration. @lukekim  - could you please confirm this should be removed or adjusted just to update tracing level but not make it verbose.

Reply:
Updated